src/dashboards/components/AutoRefresh.scss

```diff
@@ -0,0 +1,46 @@
+.refresh-form-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px 50px;
+}
+
+.refresh-form-container-child {
+  flex: 1;
+  &.active {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
+  &.inactive {
+    display: grid;
+    grid-template-columns: 1fr;
+  }
+}
+.refresh-form-column {
+  display: grid;
+}
+
+.refresh-timeout-span {
+  flex: 1;
+}
+
+.refresh-form-buttons {
+  display: grid;
+  grid-template-columns: auto auto;
+  grid-gap: 10px;
+  padding: 30px 10px 15px 10px;
+}
+
+.refresh-form-cancel-button {
+  width: 60%;
```

Comment:
i'd shy away from relative button sizes in this respect and use `@media` to set specific widths based on the screen size. The main reason being that it can be pretty unpredictable. I don't currently have context on where this exists, but I don't imagine that the button would need to be more than 200px wide (i think that's the `submit` button size that we set)